src/tests/api/cars-contollers.spec.js

```diff
@@ -0,0 +1,102 @@
+import {expect, test} from "@playwright/test";
+import {USER} from "../../data/users.js";
+import CarModel from "../../../models/carModel.js";
+import {DEFAULT_BRANDS_RESPONSE_BODY} from "../fixtures/brands.js";
+import {DEFAULT_BRAND_MODELS, DEFAULT_MODELS_RESPONSE_BODY} from "../fixtures/models.js";
+import ApiClient from "../../client/ApiClient.js";
+import {FIRST_USER_CAR_RESPONSE_BODY, userCarIDUpdate} from "../fixtures/cars.js";
+
+test.describe('Cars', ()=>{
```

Comment:
separate tests by creating test file for each test group (GET, POST, PUT, DELETE)

Reply:
fixed within https://github.com/AndreyTests/Playwright/pull/4